generate/seccomp/seccomp_default_unsupported.go

```diff
@@ -0,0 +1,14 @@
+// +build !linux
+
+package seccomp
+
+// These are copied from linux/amd64 syscall values, as a reference for other
+// platforms to have access to
```

Comment:
If these values were amd64-specific, that would be a problem, because folks generating configs for other arches from non-Linux systems would get the wrong values.  But it turns out that these values are [arch-independent][1].  Maybe adjust this comment to point that out, possibly linking to the kerrnel?

[1]: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git/tree/include/uapi/linux/sched.h?h=v4.14.15#n5